year-pages/2019-conferences-list.md

```diff
@@ -2,6 +2,7 @@
 
 | Conference | Date | Venue | Description | Scholarship |
 |------------|------|-------|-------------|-------------|
+| [React India](https://www.reactindia.io/) | 22-24 Aug | Goa | A 3 day conference with first day focussing on workshops & the next 2 days talks on things around React, React Native & GraphQL. Regular and lightning talks will cover various React.JS and FE topics including React Native, GraphQL, VR & AR, Redux, Preact and more! | No |
```

Comment:
Can you edit the description to `A 3 day conference with the first day focussing on workshops & the next 2 days on talks on React, React Native, GraphQL, VR & AR, Redux, Preact and more.`?

This way it includes all topics and is more concise. :)